src/modules/navigator/navigator_main.cpp

```diff
@@ -1040,7 +1034,6 @@ void Navigator::geofence_breach_check()
 
 				rep->current.timestamp = hrt_absolute_time();
 				rep->current.yaw = get_local_position()->heading;
```

Comment:
```suggestion
				rep->current.yaw = NAN;
```

Reply:
done in 81e2662fea71d1f95f113de51c8f381472310a05